install-gocd-plugins

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+yell() { echo "$0: $*" >&2; }
+die() { yell "$*"; exit 111; }
+try() { echo "$ $@" 1>&2; "$@" || die "cannot $*"; }
+
+# do that aslong we are not sure this is really set in the base image
+GOCD_PLUGIN_EXTERNAL_HOME=${GOCD_PLUGIN_EXTERNAL_HOME:-/godata/plugins/external}
```

Comment:
GOCD_PLUGIN_EXTERNAL_HOME - Can you please explain the comment about why we provide this when the go server will always look in /godata/plugins/external ? From the SystemEnvironment, I see that the external path is set as a system property.